common/config/rush/pnpm-lock.yaml

```diff
@@ -6423,6 +6426,7 @@ packages:
 
   /@types/debug@4.1.8:
     resolution: {integrity: sha512-/vPO1EPOs306Cvhwv7KfVfYvOJqA/S/AXjaHQiJboCZzcNDb+TIJFN9/2C9DZ//ijSKWioNyUxD792QmDJ+HKQ==}
+    requiresBuild: true
```

Comment:
These lines were added when I ran `rush update --recheck` with no other changes.  So they appear unrelated to my change, but I needed to run `rush update --recheck` to upgrade `trim`.